.github/workflows/push-image.yml

```
@@ -0,0 +1,37 @@
+name: push-image
+
+on:
+  push:
+    branchs: 
```

actionlint (1.7.4)

unexpected key "branchs" for "push" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)